Solutions/Marain.ContentManagement.Api.Services/Marain.ContentManagement.Api.Services.csproj

```diff
@@ -16,18 +20,27 @@
     <PackageReleaseNotes>
     </PackageReleaseNotes>
   </PropertyGroup>
-  <Import Project="..\Common.NetStandard_2_0.proj" />
+  
+  <ItemGroup>
+    <None Remove="Marain\Cms\Api\ContentManagementServices.yaml" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Marain\Cms\Api\Services\ContentManagementServices.yaml" />
+  </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Menes.Abstractions" Version="0.9.0" />
+    <PackageReference Include="Menes.Abstractions" Version="0.10.0-preview.8" />
```

Comment:
That's a pre-release dependency? Do we mean that?

Reply:
Yes, in that we need changes that were introduced in that specific version. We could do a proper release of Menes if we thought it valuable, but given that this library is a way off being "production ready" I don't think it's a big deal right now.

Reply:
The approach I've applied to `Corvus.Monitoring` has been that we want to have something in production without incident for at least a week as one of the requirements for saying we're done. See https://github.com/corvus-dotnet/Corvus.Monitoring/issues/33

And the idea is only to do a non-preview release once all boxes are checked.

(What's less clear right now is the right mechanism for tracking this at a higher level. I used a milestone there, but it's possible projects are a better way. Also, it's not clear how to handle multiple changes in flight at once, other than trying to avoid having too much in flight at once.)